hail_scripts/v02/load_dataset_to_es.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python
+
+import os
+
+# os.system("pip install elasticsearch")
+
+import argparse
+import hail as hl
+import logging
+from pprint import pprint, pformat
+import time
+import sys
+
+from hail_scripts.shared.elasticsearch_utils import ELASTICSEARCH_INDEX, \
+    ELASTICSEARCH_UPDATE, ELASTICSEARCH_UPSERT
+from hail_scripts.v01.utils.add_hgmd import add_hgmd_to_vds
+from hail_scripts.v01.utils.add_eigen import add_eigen_to_vds
+from hail_scripts.v01.utils.gcloud_utils import delete_gcloud_file
+from hail_scripts.v02.utils.vds_utils import read_in_dataset
+from hail_scripts.v01.utils.computed_fields import get_expr_for_variant_id, \
+    get_expr_for_vep_gene_ids_set, get_expr_for_vep_transcript_ids_set, \
+    get_expr_for_vep_consequence_terms_set, \
+    get_expr_for_vep_sorted_transcript_consequences_array, \
+    get_expr_for_worst_transcript_consequence_annotations_struct, get_expr_for_end_pos, \
+    get_expr_for_xpos, get_expr_for_contig, get_expr_for_start_pos, get_expr_for_alt_allele, \
+    get_expr_for_ref_allele, get_expr_for_vep_protein_domains_set, get_expr_for_variant_type, \
+    get_expr_for_filtering_allele_frequency
+from hail_scripts.v01.utils.elasticsearch_utils import VARIANT_GENOTYPE_FIELDS_TO_EXPORT, \
+    VARIANT_GENOTYPE_FIELD_TO_ELASTICSEARCH_TYPE_MAP, \
+    SV_GENOTYPE_FIELDS_TO_EXPORT, SV_GENOTYPE_FIELD_TO_ELASTICSEARCH_TYPE_MAP
+from hail_scripts.v01.utils.add_combined_reference_data import add_combined_reference_data_to_vds
+from hail_scripts.v01.utils.add_primate_ai import add_primate_ai_to_vds
+from hail_scripts.v01.utils.hail_utils import create_hail_context, stop_hail_context
+from hail_scripts.v01.utils.validate_vds import validate_vds_genome_version_and_sample_type, validate_vds_has_been_filtered
+from hail_scripts.v01.utils.elasticsearch_client import ElasticsearchClient
+from hail_scripts.v01.utils.fam_file_utils import MAX_SAMPLES_PER_INDEX, compute_sample_groups_from_fam_file
+from hail_scripts.v01.utils.vds_schema_string_utils import convert_vds_schema_string_to_annotate_variants_expr
+from hail_scripts.v01.utils.add_1kg_phase3 import add_1kg_phase3_to_vds
+from hail_scripts.v01.utils.add_cadd import add_cadd_to_vds
+from hail_scripts.v01.utils.add_dbnsfp import add_dbnsfp_to_vds
+from hail_scripts.v01.utils.add_clinvar import add_clinvar_to_vds
+from hail_scripts.v01.utils.add_exac import add_exac_to_vds
+from hail_scripts.v01.utils.add_gnomad import add_gnomad_to_vds
+from hail_scripts.v01.utils.add_topmed import add_topmed_to_vds
+from hail_scripts.v01.utils.add_mpc import add_mpc_to_vds
```

Comment:
Essentially, yes.

So the context is we decided to just change the minimal possible (only the hail diff), so copied and pasted everything from v01 to this. But instead of doing a PR with the whole file (since we can't see the PR diff changes in a new file),I decided to take everything out of that are the later steps so the PR is more manageable.

I left it here so as I add the other steps I won't have to worry about imports, and then when it's all done I'll remove the unused ones.

tldr; a middle ground between a way of sanity for me so I don't have to look at all the code at once, and a way for PR reviewers to not have to look at all of the non-relevant v01 code at once.

Reply:
Now that I'm thinking about it, it might make sense for me to get a copied and pasted version of v01 to v02 merged to master, and all subsequent diffs on v02 will only see v02 changes against a v01 copy.